docs/UserGuide.md

```diff
@@ -476,3 +533,63 @@ Parameter | Prefix | Constraints, Examples
 **TAG** | `t/` | Tags names should be alphanumeric and have a maximum of 45 characters. <br> e.g. `t/important`
 **SUGGESTION_TYPE** | `by/` | Suggestion types can only be either `available`, `contract` or `frequency`. <br> e.g. `by/available`
 **INDEX** | - | Index is a number greater than 0 that is based on the numberings beside each client or note. <br> e.g. `1` would refer to the first client or note.
+
+--------------------------------------------------------------------------------------------------------------------
+## Appendix A - Introduction to the UI
+
+### Start Page
+
+The diagram below shows the key UI elements of TBM upon first start-up.
+
+![Annotated UI Default Page](images/command-screenshots/UI_annotated.png)
+
+* **Command Box** allows you to input  commands in their correct syntax to interact with TBM.
+
+* **Client List Panel** will always allow you to view your client's information at a go. It's scrollable so that you 
+never have to worry about not being able to see all your important clients at a single glance!
+
+* **Client Card** will show you brief information a particular client. 
+
+* **Result Display** will show you the output of executing your command.
+    * It indicates successful commands.
+    
+    * On the off-chance there's a syntax issue, TBM will remind you of the correct syntax it expects, something like this:
+ 
+     ![Result Display slowing error message](images/command-screenshots/result_display_error_syntax.png)
+     
+* **Generic Widget View Box** is aptly named because it shall spawn *Client Cards* or *Country Cards* when
```

Comment:
I don't think spawn is the right word to use here. Also, we shouldn't put `generic widget view box` in our user guide. Something like display panel would be more appropriate

Reply:
changed to Render and changed annotation